City-of-Helsinki/drupal-helfi-platform-config
# UHF-X Translation updates

How to test: 
- Run `composer require drupal/helfi_platform_config:dev-UHF-X_translation_updates`
- Run `make new`
- Run `make drush-cex`
- Check if there is LESS overridden configuration files. 🤞
